src/core-ownership.js

```diff
@@ -9,9 +9,13 @@ import mapObject from 'map-obj'
 import { discoveryKey } from 'hypercore-crypto'
 import pDefer from 'p-defer'
 import { NAMESPACES } from './constants.js'
-
 /**
- * @typedef {import('./types.js').CoreOwnershipWithSignatures} CoreOwnershipWithSignatures
+ * @import {
+ *   CoreOwnershipWithSignatures,
+ *   CoreOwnershipWithSignaturesValue,
+ *   KeyPair,
+ *   Namespace
+ * } from './types.js'
```

Comment:
I was adding an import from `types.js` so I figured I'd fix _all_ the imports in this file. I did that in several other places too.